BitmovinAdobeAnalytics/README.md

```diff
@@ -1,3 +1,123 @@
+
+```
+pod install
+```
+
+2. Add Adobe Media Analytics to your application. Please refer to [Adobe documentation](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-media-analytics#add-media-analytics-to-your-app)
+
+- To add the Media library and its dependencies to your project, add the following pods to your Podfile:
+
+```
+pod 'ACPCore', '~> 2.0'
+pod 'ACPAnalytics', '~> 2.0'
+pod 'ACPMedia', '~> 2.0'
+```
+
+- In Xcode project, import Media extension.
+
+```swift
+import ACPMedia;
+```
+
+3. Register Media with mobile core. Please refer to [Adobe documentation](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-media-analytics#register-media-with-mobile-core)
+
+- In your app's application:didFinishLaunchingWithOptions, register Media with Mobile Core:
+
+```swift
+import ACPCore
+import ACPAnalytics
+import ACPIdentity
+import ACPMedia
+
+func application(_ application: UIApplication,
+                 didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+    ACPCore.setLogLevel(.debug)
+    ACPCore.configure(withAppId: "your-launch-app-id")
+
+    ACPAnalytics.registerExtension()
+    ACPIdentity.registerExtension()
+    ACPMedia.registerExtension()
+
+    ACPCore.start {
+
+    }
+
+    return true;
+}
+```
+
+`your-launch-app-id` : ApplicationId setup following steps [here](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/configuration)
+
+### Compatibility
+This version of the Adobe Analytics Integration is validated with `BitmovinPlayer` version `>= 2.57.x`. It should work with older versions as well but is subject to validation.
+
+## Usage
+----------------
+The BitmovinAdobeAnalytics module provides information for each video uniquely. The module exposes following objects.
+
+1. `AdobeMediaAnalytics`: This object is the entry point to be used by the application to start tracking Adobe Media Analytics. The following example shows how to setup the BitmovinAdobeAnalytics.
+
+```swift
+let adobeConfig = AdobeConfiguration()
+adobeConfig.debugLoggingEnabled = true
+do {
+    adobeAnalytics = try AdobeMediaAnalytics(player: player!,
+                                            config: adobeConfig,
+                                            delegate: self)
+} catch {
+    NSLog("[ Example ] AdobeAnalytics initialization failed with error: \(error)")
+}
+```
+2. `AdobeConfiguration`: This object is used to configure the Adobe analytics module. This can be used to configure if debugging should be enabled or not for the module.
+
+3. `AdobeAnalyticsDataOverrideDelegate`: This defines a protocol with menthods for calculating metadata to be sent with Adobe analytics events. A default implentation is provide by the module but it SHOULD be overridden by application to provide custom override methods for the values of fields of Adobe Media Analytics event data. A list of methods in the protocol and default values is provided below.
+
+##### Media
```

Comment:
I think `###` should be sufficient, the headline above is `##`. If adapted, the `Ads` section below should be changed as well

Reply:
Done